app/jobs/link_check_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class LinkCheckJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    fundings = Funding.where.not(link: nil)
+    fundings.each do |funding|
+      check_link(funding)
+      sleep 1
+    end
+  end
+
+  private
+
+  def check_link(funding)
+    response = http_request(funding.link)
+    return if response.is_a? Net::HTTPSuccess
```

Comment:
Mhh, could be 302, 307 or 308 as well, which would not really indicate a broken link.

Reply:
Resolved in f8f1485